splinepy/bezier.py

```diff
@@ -4,6 +4,57 @@
 class BezierBase(spline.Spline):
     """Bezier Base. Contain extra operations that's only
     available for bezier families.
+
+    Bezier splines are a special type of spline where the basis functions have
+    a global support. The basis functions are given by the so-called
+    *Berstein polynomials*
+
+    .. math::
+            N_{i;p}(u) = \\binom{p}{i} u^i (1-u)^{p-i}
+
+    A Bezier spline itself can be seen as a map from the parametric domain
+    :math:`\\[0,1\\]^{N_{param}}` to the physical domain
```

Comment:
```suggestion
    :math:`[0,1]^{N_{param}}` to the physical domain
```
Original did not work on my machine, please confirm that this is still valid on mac.